packages/vite-plugin-checker/src/checkers/eslint/main.ts

```diff
@@ -83,6 +85,9 @@ const createDiagnostic: CreateDiagnostic<'eslint'> = (pluginConfig) => {
       }
 
       const handleFileChange = async (filePath: string, type: 'change' | 'unlink') => {
+        if (pluginConfig.eslint) {
+          pluginConfig.eslint.delay && (await sleep(pluginConfig.eslint.delay))
```

Comment:
should remove sleep now

Reply:
thx for comment.I have seen lodash.debounce in  packages/vite-plugin-checker/package.json file.i will create a new pr for debounce usage.

Reply:
Saw the new PR, however you still can edit on the original PR to keep all the comments next time :)